src/App.css

```diff
@@ -1,23 +1,61 @@
+@import url("https://fonts.googleapis.com/css?family=Bebas+Neue|Roboto+Condensed:wght@700|Scada|Squada+One|Nunito+Sans:300,600,800|Concert+One&display=swap");
```

Comment:
This is a nice touch but you might want to be a bit careful when loading lots of fonts. Font files can be quite large so they can slow your website down a lot.

Reply:
You are right @40thieves , I have to remove extra fonts, Thank you so much for review